recipes/libcurl/all/conanfile.py

```diff
@@ -144,24 +150,33 @@ def configure(self):
 
     def requirements(self):
         if self.options.with_ssl == "openssl":
-            self.requires("openssl/1.1.1g")
+            self.requires("openssl/1.1.1h")
         elif self.options.with_ssl == "wolfssl":
-            self.requires("wolfssl/4.4.0")
+            self.requires("wolfssl/4.5.0")
         if self.options.with_nghttp2:
             self.requires("libnghttp2/1.40.0")
         if self.options.with_libssh2:
             self.requires("libssh2/1.9.0")
         if self.options.with_zlib:
             self.requires("zlib/1.2.11")
         if self.options.with_brotli:
-            self.requires("brotli/1.0.7")
+            self.requires("brotli/1.0.9")
+        if self.options.get_safe("with_zstd"):
+            self.requires("zstd/1.4.5")
+
+    def package_id(self):
+        # Deprecated options
```

Comment:
I would keep the deletes in `configure`, because the `build_id` will be different for no reason when doing the remove in `package_id`.

Reply:
I thought that this error https://github.com/conan-io/conan-center-index/issues/3206 could be avoided by deleting options in package_id() instead of configure().

Reply:
Oh, I didn't foresee that problem.
Indeed, good solution. I guess we don't care about `build_id`?

Reply:
I agree, build_id doesn't matter I guess